client/components/plans/plan-icon/index.jsx

```diff
@@ -0,0 +1,497 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import {
+	PLAN_FREE,
+	PLAN_PREMIUM,
+	PLAN_BUSINESS,
+	PLAN_JETPACK_FREE,
+	PLAN_JETPACK_BUSINESS,
+	PLAN_JETPACK_BUSINESS_MONTHLY,
+	PLAN_JETPACK_PREMIUM,
+	PLAN_JETPACK_PREMIUM_MONTHLY,
+	PLAN_PERSONAL
+} from 'lib/plans/constants';
+
+export default class PlanIcon extends Component {
+	render() {
+		const { plan } = this.props;
+
+		return (
+			<div className="plan-icon">
```

Comment:
While I added a class name here `plan-icon`, I intentionally didn't put any styling to PlanIcon because we use different sizes of plans icons in different places.

Maybe we could add some reasonable defaults. What do you think?